src/imgui/Shortcuts.cc

```diff
@@ -0,0 +1,60 @@
+#include "Shortcuts.hh"
+#include "imgui_internal.h"
+
+#include "SettingsConfig.hh"
+
+#include <array>
+
+namespace openmsx {
+
+static constexpr auto defaultShortcuts = std::array{
+	Shortcuts::Data{ShortcutIndex::GOTO_MEMORY_ADDRESS, ImGuiMod_Ctrl | ImGuiKey_G, ShortcutType::LOCAL},
+	Shortcuts::Data{ShortcutIndex::GOTO_DISASM_ADDRESS, ImGuiMod_None, ShortcutType::LOCAL},
+};
+
+Shortcuts::Shortcuts(SettingsConfig& config_)
+	: config(config_)
+	, shortcuts{}
+{
+	config.setShortcuts(*this);
+	loadInit();
+}
+
+void Shortcuts::loadInit()
+{
+	for (auto& shortcut : defaultShortcuts) {
+		setShortcut(shortcut.index, shortcut.keychord, shortcut.local, shortcut.repeat);
+	}
+}
+
+Shortcuts::Data& Shortcuts::getShortcut(ShortcutIndex index)
+{
+	assert(index < ShortcutIndex::NUM_SHORTCUTS);
+	return shortcuts[index];
+}
+
+void Shortcuts::setShortcut(ShortcutIndex index, ImGuiKeyChord keychord, std::optional<bool> local, std::optional<bool> repeat)
+{
+	assert(index < ShortcutIndex::NUM_SHORTCUTS);
+	shortcuts[index].index    = index;
+	shortcuts[index].keychord = keychord;
+	if (local) shortcuts[index].local = *local;
+	if (repeat) shortcuts[index].repeat = *repeat;
+}
+
+std::string Shortcuts::getShortcutName(ShortcutIndex index)
+{
+	assert(index < ShortcutIndex::NUM_SHORTCUTS);
+	static constexpr auto shortcutNames = std::array{
+		"GotoMemoryAddress", "GotoDisasmAddress"
+	};
+	return shortcutNames[index];
+}
+
+bool Shortcuts::isPressed(ShortcutIndex index)
+{
+	// what kind of flag should be tested?
+	return shortcuts[index].keychord != ImGuiKey_None;
+}
```

Comment:
This method isn't used anywhere. And i also don't see a use for it in the future? Remove it.